packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/CustomVisConfigTabs.tsx

```diff
@@ -73,69 +416,217 @@ const CustomVisConfigTabs: React.FC = memo(() => {
-                },
-            }}
-        >
-            <Tabs.List>
-                <Tabs.Tab value="config">Config</Tabs.Tab>
-                <Tabs.Tab value="data">Data</Tabs.Tab>
-            </Tabs.List>
-
-            <Tabs.Panel value="config">
-                <Editor
-                    loading={<Loader color="gray" size="xs" />}
-                    beforeMount={(monaco) =>
-                        loadMonaco(monaco, schemas.current!)
-                    }
-                    defaultLanguage="json"
-                    options={{ ...MONACO_DEFAULT_OPTIONS }}
-                    value={visSpec}
-                    onChange={(config) => setVisSpec(config ?? '')}
+        <>
+            <Flex justify="space-between" align="flex-end">
+                <SelectTemplate
+                    itemsMap={itemsMap}
+                    isCustomConfig={isCustomConfig}
+                    isEditorEmpty={isEditorEmpty}
+                    setEditorConfig={setEditorConfig}
                 />
-            </Tabs.Panel>
+                {isAiEnabled && (
+                    <>
+                        <Text>or</Text>
+                        <GenerateVizWithAi
+                            itemsMap={itemsMap}
+                            sampleResults={series.slice(0, 3)}
+                            setEditorConfig={setEditorConfig}
+                            editorConfig={editorConfig}
+                        />
+                    </>
+                )}
+                <DocumentationHelpButton href="https://docs.lightdash.com/references/custom-charts#custom-charts" />
+            </Flex>
+            {/* <Tabs
+                defaultValue="config"
+                style={{ flexGrow: 1 }}
+                styles={{
+                    root: {
+                        display: 'flex',
+                        flexDirection: 'column',
+                    },
+                    panel: {
+                        flexGrow: 1,
+                    },
+                }}
+            >
+               <Tabs.List>
+                    <Tabs.Tab value="config">Config</Tabs.Tab>
+                    <Tabs.Tab value="data">Data</Tabs.Tab>
+                   
+                </Tabs.List>
+
+                <Tabs.Panel value="config">
+                */}
+            <Group mt="sm" h="100%" align="top">
+                {/* Hack to show a monaco placeholder */}
+                {isEditorEmpty ? (
+                    <Text
+                        ml="xl"
+                        pos="absolute"
+                        w="330px"
+                        color="gray.5"
+                        sx={{
+                            pointerEvents: 'none',
+                            zIndex: 100,
+                            fontFamily: 'monospace',
+                        }}
+                    >
+                        {`Write some vega lite JSON or select a template. Check our docs for more info and examples.`}
```

Comment:
```suggestion
                        Write some vega lite JSON or select a template. Check our docs for more info and examples.
```

Reply:
(very low priority comment) I've definitely seen a placeholder in vscode. wonder if there's a way...

Reply:
I found that solution on a github issue on monaco, iirc the placeholder in vscode is hardcoded, and this was a workaround.

Reply:
ok great!